Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.11 as builder
+FROM golang:1.17.6-alpine3.15 as builder
```

Comment:
I updated this to `golang:1.17-alpine` in my update go modules PR a few days ago.

Reply:
I think I've seen it, but I did not find it. Fixed